Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_test.yml

```diff
@@ -0,0 +1,357 @@
+id: Test XDR Playbook
```

Comment:
I think this should be in TestPlaybooks folder.

Reply:
@anara123 see discussion in the issue